src/main/java/nurseybook/commons/util/StringUtil.java

```diff
@@ -28,8 +27,8 @@ public static boolean containsWordIgnoreCase(String sentence, String word) {
         requireNonNull(word);
 
         String preppedWord = word.trim();
-        checkArgument(!preppedWord.isEmpty(), "Word parameter cannot be empty");
-        checkArgument(preppedWord.split("\\s+").length == 1, "Word parameter should be a single word");
+        AppUtil.checkArgument(!preppedWord.isEmpty(), "Word parameter cannot be empty");
```

Comment:
Could you import this at the start of the file and simply use `checkArgument(...)` instead? Similarly for line 31 below.